### PR TITLE
Getting started with directus and django

### PR DESCRIPTION
**Pull request by mizzleinetimi**

### CODE REVIEW

File: directus-django/integrating directus in django.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+```bash
+python -m venv myenv
+```
+Then go ahead and activate the virtual environment:
+- On Windows: myenv\Scripts\activate
+- On macOS/Linux: source myenv/bin/activate
+
+
+#### Installing Django
```
Comment (phazonoverload) [inline directus-django/integrating directus in django.md:30]: Remove H4 - just go straight into the next step.

File: directus-django/integrating directus in django.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+Django is one of the most popular python frameworks known for its battery included philosophy, it comes loaded with a lot of features that cuts down development time for  devs. In this tutorial, you will learn how to integrate django with Directus using a blog API. At the end of this tutorial, you will have an application that uses the django templating engine to display data from the API.
+
+
+## Before You Start
+You will need:
+- Python installed and a code editor on your computer.
+- Directus project - Use the [quickstart guide](https://docs.directus.io/getting-started/quickstart) to create a project with the fields  title, content and author.
+- Basic knowledge of python and django
+
+## Project setup
```
Comment (phazonoverload) [inline directus-django/integrating directus in django.md:19]:
```suggestion
## Create a Django Project
```

File: directus-django/integrating directus in django.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+
+#### Installing Django
+With the virtual environment activated, install Django within this env:
+
+```bash
+pip install django
+```
+
+This installs django within the virtual environment.
```
Comment (phazonoverload) [inline directus-django/integrating directus in django.md:37]: Remove this - it's repeating what you say before the command

File: directus-django/integrating directus in django.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+#### Initializing Django Project
+Create a new Django project and move into the directory of the new project
+
+```bash
+django-admin startproject myblog
+cd myblog
+```
+
+This command starts a new django project and loads up the boilerplate files within django
```
Comment (phazonoverload) [inline directus-django/integrating directus in django.md:47]: Again, remove - it's repeating what you've already said

File: directus-django/integrating directus in django.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+
+To fetch data from the Directus API, we will use the *requests* library. It is not included in Python’s standard library, so it needs to be installed separately:
+
+```bash
+pip install requests
+```
+We need it to be able to make http requests to external API’s.
```
Comment (phazonoverload) [inline directus-django/integrating directus in django.md:54]: Remove

File: directus-django/integrating directus in django.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+
+
+#### Creating the Blog App:
+Create a new app within your Django project to handle the blog functionality:
+```bash
+python manage.py startapp blog
+```
+This creates a new directory within our project containing all the necessary files for a django app
```
Comment (phazonoverload) [inline directus-django/integrating directus in django.md:62]: This isn't repeating what was before, but let's consolidate it with the previous sentence before the command.

**Review by phazonoverload — CHANGES_REQUESTED** [review]
I started reviewing this but have stopped because it does not follow the exact same format as our other guides. 

It should be: 
- Initialize Django Project (to the point of running dev server)
- Set up requests/env vars/anything needed to call Directus
- Using Global Metadata and Settings 
- Creating Pages With Directus 
- Creating Blog Posts With Directus 
  - Create Blog Post Listing
  - Create Blog Post Single
- Add Navigation

Please make it match this format: https://docs.directus.io/blog/getting-started-directus-and-eleventy-11ty-3

**Comment by phazonoverload** [comment]
Any update on this please?

**Comment by mizzleinetimi** [comment]
Had a few setbacks. Will be ready on or before Thursday.

**Comment by mizzleinetimi** [comment]
Still working on this.

**Comment by phazonoverload** [comment]
Ok noted - it's been about a month since the initial review. I'm going to need amends made latest end of Thursday so I can review Friday. If that's not possible, let me know, but I may have to take it out of the content schedule

**Review by phazonoverload — CHANGES_REQUESTED** [review]
Added a couple of comments I need you to answer. I also would like you to be really explicit around where each file is meant to be placed -in what directories, etc. Every time you mention a new file, specify. 

There are some other small edits, but I'll handle those while publishing.

File: directus-django/integrating directus in django.md [hunk 7]
```diff
@@ -0,0 +1,290 @@
...
+  mkdir my_django_site && cd my_django_site
+  django-admin startproject config .
+  python -m venv env
+  source env/bin/activate  # On Windows use `env\Scripts\activate`
+  pip install django requests
+```
+After setting up your project structure, let's configure Django to serve as the foundation of your dynamic website:
+
+ Open your Django project in your code editor of choice.
+Modify settings.py to include the necessary configurations for Directus API communication.
```
[the remark this answers is not in the document]
Comment (mizzleinetimi) [reply directus-django/integrating directus in django.md]: was a placeholder oversight on my end, updated the settings properly now.

File: directus-django/integrating directus in django.md [hunk 8]
```diff
@@ -0,0 +1,290 @@
...
+    return response.json()
+
+def get_collection_items(collection):
+    response = requests.get(f"{DIRECTUS_API_ENDPOINT}/items/{collection}")
+    return response.json()
+    
+```
+
+With the functions in place, you can now fetch global settings and pass them to your Django templates.
+
```
[the remark this answers is not in the document]
Comment (mizzleinetimi) [reply directus-django/integrating directus in django.md:96]: done.

**Comment by mizzleinetimi** [comment]
made all requested changes.

**Review by phazonoverload — APPROVED** [review]

**Comment by phazonoverload** [comment]
Thank you for writing this, and I'm happy to consider it done in it's current state. I'll schedule it in to be published. 

This repo is just for authoring/reviewing - the actual piece will be published via a Directus project, so I'll take it from here. I'll close this issue to keep the main branch clean, but this is a acceptance of your work.

Could you email me at [kevin@directus.io](mailto:kevin@directus.io) and I'll share what we need to get you paid.

I'm very excited to publish this piece - it's very cool.

**Pull request closed without merging**